src/main.jsx

```diff
@@ -24,6 +24,10 @@ let routes = [
     path: "/",
     element: <Home />,
   },
+  {
+    path: "/home",
+    element: <Home />
```

Comment:
```suggestion
    element:  <RedirectToRoot />
```

Hi @Heun11, we are actually looking for a redirect. A component that simply navigates to `/` would do

```
import { createBrowserRouter, RouterProvider, Navigate } from "react-router-dom";

const RedirectToRoot = () => <Navigate to="/" />;
```

Would you mind making that change?

Reply:
Sure, I'll make new PR